store/README.md

```diff
@@ -0,0 +1,12 @@
+# Store
+
+Package `store` provides a modeling kit for decision automation problems. It is
+based on the paradigm of "decisions as code". The base interface is the
+[Store][Store]: a space defined by variables and logic. Hop is an engine
```

Comment:
We are leaving behind Hop terminology. Can we remove it from here and from any other place where it is present?